code/extensions/GoogleSitemapExtension.php

```diff
@@ -33,7 +33,8 @@ public function canIncludeInGoogleSitemap()
             $can = $this->owner->getGooglePriority();
         }
 
-        $this->owner->invokeWithExtensions('alterCanIncludeInGoogleSitemap', $can);
+        // Allow override. In this case, since we don't know if this should be an "and" vs. "or" override, we'll use the LAST result (if any).
+        if ($override = $this->owner->invokeWithExtensions('alterCanIncludeInGoogleSitemap', $can)) $can = array_pop($override);
```

Comment:
p.s. @dhensby if you'd like, I will happily drop in that `min()` you're referring to, replacing `array_pop()` since it won't affect me personally. It's more of a theoretical debate at this point :smile: